docs/known_issues/eessi-2023.06.md

```diff
@@ -25,4 +25,7 @@ export OMPI_MCA_btl='^uct,ofi'
 export OMPI_MCA_pml='ucx'
 export OMPI_MCA_mtl='^ofi'
 ```
+
+You may also set these additional environment variables via site-specific Lmod hooks. For more information about how to write and implement site-specific Lmod hooks, please check  [EESSI Site Specific Configuration LMOD Hooks](site_specific_config/lmod_hooks.md)
```

Comment:
The documentation you reference here is general documentation on how to create a hook. However, I would give the specific implementation of the hook that would fix this here as well, i.e.

````suggestion
You may also set these additional environment variables via site-specific Lmod hooks:
```
require("strict")
local hook=require("Hook")

-- Fix Failed to modify UD QP to INIT on mlx5_0: Operation not permitted
function set_my_env_var_openmpi(t)
    local simpleName = string.match(t.modFullName, "(.-)/")
    if simpleName == 'OpenMPI' then
        setenv('OMPI_MCA_btl', '^uct,ofi')
        setenv('OMPI_MCA_pml', 'ucx')
        setenv('OMPI_MCA_mtl', '^ofi')
    end
end

local function combined_load_hook(t)
    if eessi_load_hook ~= nil then
        eessi_load_hook(t)
    end
    set_my_env_var_openmpi(t)
end

hook.register("load", combined_load_hook)
```
 For more information about how to write and implement site-specific Lmod hooks, please check  [EESSI Site Specific Configuration LMOD Hooks](site_specific_config/lmod_hooks.md)
````